.gitignore

```diff
@@ -1 +1,2 @@
 vendor
+.idea
```

Comment:
this is specific to your dev env and I think it belongs in you root gitignore.
```suggestion
```

I have this in my $HOME/.gitconfig
```gitconfig
[core]
	excludesfile = /Users/azr/.gitignore
```

Reply:
oh thanks for the tip!!